src/facades/management.ts

```diff
@@ -0,0 +1,40 @@
+import { Inner } from "../inner";
+import {
+  AssetController,
+  ContractAgreementController,
+  ContractDefinitionController,
+  ContractNegotiationController,
+  DataplaneController,
+  PolicyDefinitionController,
+  TransferProcessController,
+} from "../controllers/management-controllers";
+
+export class ManagementController {
+  #inner: Inner;
+
+  constructor(inner: Inner) {
+    this.#inner = inner;
+  }
+
+  get assetController() {
```

Comment:
I would drop the `Controller` word from these getters, so then the expression to, e.g. create an asset would be:
`management.asset.create(...)`. 
Maybe also adding the  `s` could reflect the "rest resource" concept, becoming:
`management.assets.create(...)`